src/main/antora/modules/web/pages/servlets/servlets003.adoc

```diff
@@ -1,58 +1,158 @@
 == Sharing Information
 
-Web components, like most objects, usually work with other objects to accomplish their tasks.
-Web components can do so by doing the following.
-
-* Using private helper objects (for example, JavaBeans components).
-
-* Sharing objects that are attributes of a public scope.
-
-* Using a database.
-
-* Invoking other web resources.
-The Jakarta Servlet technology mechanisms that allow a web component to invoke other web resources are described in xref:servlets/servlets.adoc#_invoking_other_web_resources[Invoking Other Web Resources].
-
-=== Using Scope Objects
-
-Collaborating web components share information by means of objects that are maintained as attributes of four scope objects.
-You access these attributes by using the `getAttribute` and `setAttribute` methods of the class representing the scope.
-<<_scope_objects>> lists the scope objects.
-
-[[_scope_objects]]
+Web components, like most objects, usually work with data to accomplish their tasks.
+Web components can store this information in a database or in a scoped bean.
```

Comment:
"This data can be stored in a database or a scoped bean." (to about reusing "Web Components").

Shouldn't we also mention other types of objects, though?

Reply:
https://github.com/jakartaee/jakartaee-tutorial/pull/36/commits/080bea3629fe01f40eac75638f8b5a1a1c8310af